osbs-client.spec

```diff
@@ -155,12 +156,13 @@ This package contains osbs Python 3 bindings.
 %py3_install
 mv %{buildroot}%{_bindir}/osbs %{buildroot}%{_bindir}/osbs-%{python3_version}
 ln -s  %{_bindir}/osbs-%{python3_version} %{buildroot}%{_bindir}/osbs-3
-%endif # with_python3
-
+ln -s  %{_bindir}/osbs-%{binaries_py_version} %{buildroot}%{_bindir}/osbs
```

Comment:
Looks like this line (along with line 164) could be moved outside of the %if.

Reply:
ok